cves/kernel/CVE-2020-14386.yml

```diff
@@ -137,10 +150,15 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    I could not find any unit tests testing for the value of the variable that
+    was vulnerable to overflow.
+  fix: false
+  fix_answer: |
+    There was not an automated unit test created after the fix. The fix
+    just added a quick conditional check to ensure the variable does not
```

Comment:
Looking at the fix, it looks like the overflow will still happen, they just added the check to make sure it is caught before the macoff value is calculated. Can you check this and possibly revise the wording here?